internal/utils/command_yq.go

```diff
@@ -26,8 +26,8 @@ var (
 
 var cmdYq = &cobra.Command{
 	Use:   "yq",
-	Short: "Parse json strings",
-	Long: `Parse json strings
+	Short: "Parse yaml strings",
+	Long: `Parse yaml strings
 
 The yq subcommand makes use of gojq (https://github.com/itchyny/gojq) to provide
```

Comment:
Not sure about this. @sanderblue Do you know if that's correct?

Reply:
@kidk This is correct, it seems we copied contents from `command_jq` when yq was added, just missed this